recipes/hapog/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+export C_INCLUDE_PATH=${PREFIX}/include
+export LIBRARY_PATH=${PREFIX}/lib
+
+python setup.py install --single-version-externally-managed --record=record.txt
```

Comment:
Looks like Python modules might be installed as well as the binary, if so they should be tested with an [import test](https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#python-imports) in the test section.